kolibri/content/fixtures/content_import_test.json

```diff
@@ -0,0 +1 @@
":"content.language"},{"fields":{"lang_code":"zh","lang_subcode":"02","lang_name":"Chinese-Test","lang_direction":"ltr"},"pk":2,"model":"content.language"},{"fields":{"lang_code":"es","lang_subcode":"03","lang_name":"Spanish-Test","lang_direction":"ltr"},"pk":3,"model":"content.language"},{"fields":{"lang":1,"local_file":"9f9438fe6b0d42dd8e913d7d04cfb2b2","thumbnail":false,"available":true,"preset":"high_res_video","contentnode":"32a941fb77c2576e8f6b294cde4c3b0c","supplementary":false},"pk":"9f9438fe6b0d42dd8e913d7d04cfb2b1","model":"content.file"},{"fields":{"lang":1,"local_file":"725257a0570044acbd59f8cf6a68b2be","thumbnail":false,"available":true,"preset":"low_res_video","contentnode":"32a941fb77c2576e8f6b294cde4c3b0c","supplementary":false},"pk":"725257a0570044acbd59f8cf6a68b2bf","model":"content.file"},{"fields":{"lang":3,"local_file":"e00699f859624e0f875ac6fe1e13d648","thumbnail":false,"available":true,"preset":"vector_video","contentnode":"2b6926ed22025518a8b9da91745b51d3","supplementary":false},"pk":"e00699f859624e0f875ac6fe1e13d649","model":"content.file"},{"fields":{"lang":null,"local_file":"4c30dc7619f74f97ae2ccd4fffd09bf2","thumbnail":false,"available":true,"preset":"low_res_video","contentnode":"2b6926ed22025518a8b9da91745b51d3","supplementary":false},"pk":"4c30dc7619f74f97ae2ccd4fffd09bf3","model":"content.file"},{"fields":{"lang":null,"local_file":"8ad3fffedf144cba9492e16daec1e39a","thumbnail":false,"available":true,"preset":"caption","contentnode":"2b6926ed22025518a8b9da91745b51d3","supplementary":true},"pk":"8ad3fffedf144cba9492e16daec1e39b","model":"content.file"},{"fields":{"file_size":null,"available":true,"extension":"mp4"},"pk":"9f9438fe6b0d42dd8e913d7d04cfb2b2","model":"content.localfile"},{"fields":{"file_size":null,"available":true,"extension":"mp4"},"pk":"725257a0570044acbd59f8cf6a68b2be","model":"content.localfile"},{"fields":{"file_size":null,"available":true,"extension":""},"pk":"e00699f859624e0f875ac6fe1e13d648","model":"content.localfile"},{"fields":{"file_size":null,"available":true,"extension":"mp3"},"pk":"4c30dc7619f74f97ae2ccd4fffd09bf2","model":"content.localfile"},{"fields":{"file_size":null,"available":true,"extension":"vtt"},"pk":"8ad3fffedf144cba9492e16daec1e39a","model":"content.localfile"},{"fields":{"license_name":"WTFPL"},"pk":1,"model":"content.license"},{"fields":{"license_name":"GNU"},"pk":2,"model":"content.license"},{"fields":{"license_name":"CC"},"pk":3,"model":"content.license"},{"fields":{"license_name":"MIT"},"pk":4,"model":"content.license"},{"fields":{"name":"testing","description":"test data","thumbnail":"","author":"eli","version":0,"min_schema_version":"1","root":"da7ecc42e62553eebc8121242746e88a"},"pk":"6199dde695db4ee4ab392222d5af1e5c","model":"content.channelmetadata"},{"fields":{"contentnode":"2b6926ed22025518a8b9da91745b51d3","assessment_item_ids":"[\"test\"]","number_of_assessments":1,"mastery_model":"{}"},"pk":"6199dde695db4ee4ab392222d5af1e5c","model":"content.assessmentmetadata"}]
```

Comment:
Duplicated this here, so as to have a unique fixture for the content_import tests, so that we don't have to update the schema reflection data jsons just because the base `content_test.json` is updated (as it has been by @ralphiee22 on 0.4.x).